pkgs/applications/science/biology/obitools/obitools3.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchurl, python3Packages, cmake, python3 }:
+
+let
+  pythonPackages = python3Packages;
+in
+
+pythonPackages.buildPythonApplication rec {
+  pname = "obitools3";
```

Comment:
Is obitools3 a separate program than "obitools" https://repology.org/project/obitools/versions ?

Reply:
I confirmed for myself that it is on your gitlab instance.